src/main.js

```diff
@@ -35,23 +34,15 @@ let load = loader({
 
   monitor: {
     requires: ['cfg', 'drain'],
-    setup: ({cfg, drain}) => base.stats.startProcessUsageReporting({
-      drain:      drain,
-      component:  cfg.app.statsComponent,
-      process:    'server'
+    setup: ({cfg, drain}) => base.monitor({
+      project:      'taskcluster-auth',
+      drain:        drain,
+      credentials:  cfg.taskcluster.credentials,
```

Comment:
Here is the problem I was mentioning earlier. I'm fairly sure this won't be available in the config because this is auth itself. This may require us to modify `tc-lib-monitor` somewhat. I'm not sure yet.